pattern_library/static/pattern_library/src/js/components/syntax-highlighting.js

```diff
@@ -1,9 +1,9 @@
 import hljs from 'highlight.js/lib/core';
+import django from 'highlight.js/lib/languages/django';
+import yaml from 'highlight.js/lib/languages/yaml';
 
 export default function() {
-    hljs.initHighlightingOnLoad();
-    ['django', 'yaml'].forEach((langName) => {
-        const langModule = require(`highlight.js/lib/languages/${langName}`);
-        hljs.registerLanguage(langName, langModule);
-    });
+    hljs.registerLanguage('django', django);
+    hljs.registerLanguage('yaml', yaml);
+    hljs.highlightAll();
 }
```

Comment:
With this change, it seems the webpack.config.js overrides for highlight.js are now irrelevant.